anchor/common/ssv_types/src/share.rs

```diff
@@ -0,0 +1,8 @@
+use types::PublicKey;
+
+/// One of N shares of a split validator key.
+#[derive(Debug, Clone)]
+pub struct Share {
+    /// The public key of this Share
+    pub share_pubkey: PublicKey,
```

Comment:
would we not need operatorID here?

Reply:
The `Share` is part of `ClusterMember`, and the operator ID can be found there

Reply:
Something I considered but this is also a it depends on the structure of the database. We do have the access to the OperatorID from the ClusterMember. My idea was to go from ClusterID => Share where the Share is owned by the current operator, so storing the ID on the share is not needed in this case.